blocks/top-table-list-block/features/top-table-list/default.scss

```
@@ -1,129 +1,128 @@
 .top-table-list-section-extra_large {
-  .xl-large-promo {
-    .xl-promo-headline {
-      @media screen and (min-width: map-get($grid-breakpoints, 'md')) {
-        margin-bottom: map-get($spacers, 'xs');
-      }
-    }
-  }
+	.xl-large-promo {
+		.xl-promo-headline {
+			@media screen and (min-width: map-get($grid-breakpoints, "md")) {
+				margin-bottom: map-get($spacers, "xs");
+			}
+		}
+	}
 }
 
-
 .top-table-list-section-large {
-  .large-promo {
-    .lg-promo-headline {
-      padding-top: map-get($spacers, 'xs');
-      @media screen and (min-width: map-get($grid-breakpoints, 'md')) {
-        padding-top: 0;
-        margin-top: - map-get($spacers, 'xs');
-      }
-    }
-  }
+	.large-promo {
+		.lg-promo-headline {
+			padding-top: map-get($spacers, "xs");
+			@media screen and (min-width: map-get($grid-breakpoints, "md")) {
+				padding-top: 0;
+				margin-top: -map-get($spacers, "xs");
```

This is causing an regression shown in chromatic, we will need to wrap the `map-get()` call in `()` 
```suggestion
				margin-top: -(map-get($spacers, "xs"));
```

It's a known issue with prettier - https://arcpublishing.atlassian.net/wiki/spaces/TI/pages/3015311539/Component+Nomenclature+Status

---

wrong link?

---

Updated - sorry about that

---

no worries, had found it. updated. no ui regressions (thank you storybook/chromatic)